pkg/skaffold/runner/runner.go

```diff
@@ -172,7 +172,29 @@ func (r *SkaffoldRunner) Run(ctx context.Context, out io.Writer, artifacts []*v1
 		return errors.Wrap(err, "deploy step")
 	}
 
-	return nil
+	if !r.opts.Tail {
+		return nil
+	}
+	// If tail is true, stream logs from deployed objects
+	imageList := kubernetes.NewImageList()
+	for _, build := range bRes {
+		imageList.Add(build.Tag)
+	}
+	colorPicker := kubernetes.NewColorPicker(artifacts)
+	logger := kubernetes.NewLogAggregator(out, imageList, colorPicker)
+
+	if err := logger.Start(ctx); err != nil {
+		return errors.Wrap(err, "starting logger")
+	}
+
+	for {
```

Comment:
Could this be replaced by just `<-ctx.Done()`?

Reply:
Also, should we setup the signal handling to handle ctrl-c properly?

Reply:
Oh yah good point, I'll replace it with `<-ctx.Done()`

For handling ctrl-c, I wasn't sure what the benefit of setting it up would be since there isn't any cleanup that needs to happen, the log stream just needs to end.

Reply:
Yeah, same for me. I think it's cleaner but I have no idea if it's really better

Reply:
Maybe we can add the ctrl-c handler to every command in another PR

Reply:
Sounds good!